include/dpp/dispatcher.h

```diff
@@ -715,6 +715,33 @@ struct integration_delete_t : public event_dispatch_t {
 	integration deleted_integration;
 };
 
+/** */
+struct thread_create_t : public event_dispatch_t {
+	/**
+	 */
+	thread_create_t(class discord_client* client, const std::string& raw);
+	guild* creating_guild;
+	channel* created;
```

Comment:
should these be channel* and not just channel? as you're putting 't' directly in them in the events, which is a non-ptr type?